openrtx/include/interfaces/radio.h

```diff
@@ -55,6 +55,14 @@ void radio_init(const rtxStatus_t *rtxState);
  */
 void radio_terminate();
 
+void radio_initVocoder();
```

Comment:
I think is better to move these API functions to a dedicated module and/or inside the OpMode DMR handler.
As I said above, the "radio" interface is meant primarily to handle only the management of the baseband section.